### DISCUSSION TITLE
开发版关于 “1.9.3” 的讨论和 Bug 汇集放这里吧

### DISCUSSION THREAD

**Discussion by maboloshi** [maboloshi/github-chinese#278] [General]
开发版关于 “1.9.3” 的讨论和 Bug 汇集放这里吧

**Comment by TC999** [comment]
可惜之前的评论全无了（

**Comment by wyc-26** [comment]
没事，我号解了之后就会有了

**Comment by wyc-26** [comment]
那个 issue 现在只有我能看的到，只好给大家发截图了
<details><summary>全部图片（⚠ 图片超多，加载可能稍慢）</summary>
<p>

![image](https://github.com/user-attachments/assets/40da0e6a-8556-4baa-908d-834458534ea9)

![image](https://github.com/user-attachments/assets/77c8457d-77a5-4f0e-aa15-299855d7adc8)

![image](https://github.com/user-attachments/assets/9ce0772b-cd1c-4d78-82e1-9961231a42b9)

![image](https://github.com/user-attachments/assets/abc1272a-6c94-4fe0-8804-ce874884b990)

![image](https://github.com/user-attachments/assets/219f880f-7166-4114-a3f6-6e35bbff034e)

![image](https://github.com/user-attachments/assets/8f861628-caf5-405d-9fa1-d41ae49d53bc)

![image](https://github.com/user-attachments/assets/fe9e46b0-e587-4004-a5d0-fed4905b02cd)

![image](https://github.com/user-attachments/assets/19b5b9cd-1510-41ec-9158-eab4e3a6cbeb)

![image](https://github.com/user-attachments/assets/f4758a78-9ca3-4633-b3d1-6444028fdd74)

![image](https://github.com/user-attachments/assets/2b84a1a6-4b62-46d3-87ca-64bc86cf19d9)

![image](https://github.com/user-attachments/assets/ece3f51b-46d3-4710-8dc4-32c63c0896ca)

![image](https://github.com/user-attachments/assets/8883dad8-325d-467d-8799-a51a62be3fcb)

![image](https://github.com/user-attachments/assets/5023e299-a93c-42ea-b0a9-5e57850bb8ab)

![image](https://github.com/user-attachments/assets/8eabc185-94da-41dd-8e32-7b6a08a48519)

![image](https://github.com/user-attachments/assets/060a2dd8-be39-4e7e-8605-674f290872f3)

</p>
</details>

**Comment by TC999** [comment]
提交页日期消失（"日"及数字消失）
![image](https://github.com/user-attachments/assets/b3dd905e-b6cc-427e-af39-b503fa727e2f)

**Reply by maboloshi** [reply]
收到 时间正则是存在问题

**Reply by maboloshi** [reply]
本地已修正 
![image](https://github.com/user-attachments/assets/b012f71d-bbb8-4759-b8cf-ca9bb903c4bb)

**Comment by TC999** [comment]
这句话死活不翻译，已经加了词条
![image](https://github.com/user-attachments/assets/7e8d5a09-1e37-496e-b8af-34c8fc0f19c4)

示例链接：#280

**Reply by maboloshi** [reply]
"This issue was","closed" 分2个词条

**Reply by wyc-26** [reply]
PR 部分也有类似的提示，记得加一下
~GitHub 这次更新都不说一声~

**Reply by TC999** [reply]
这个词条不用管了，已经消失了

**Comment by wyc-26** [comment]
@maboloshi 可以帮忙把 `patch` 分支合并到主分支吗
（其实我是交了个 #281 的，但由于众所周知的原因看不到）

**Reply by maboloshi** [reply]
`Pull request creation failed. Validation failed: A pull request already exists for maboloshi:patch.`

**Reply by maboloshi** [reply]
这个看不到的 #281 占用了没法创建新的PR

**Reply by wyc-26** [reply]
懂了，我得先关掉 #281

**Reply by wyc-26** [reply]
done，现在再试试

**Reply by maboloshi** [reply]
关了也不行 要删除

**Reply by wyc-26** [reply]
问题是没权限删啊 🥲

**Reply by wyc-26** [reply]
算了，先不用搞了
等账户解封再合并也不是不行，只是不知道还要多久……

**Comment by wyc-26** [comment]
<details><summary>图片</summary>
<p>

![image](https://github.com/user-attachments/assets/4e92834a-5179-44db-a875-f3f5169d992c)

![image](https://github.com/user-attachments/assets/00e25e3a-4be0-4f6c-a9a7-81a6e8533910)

</p>
</details>

我加了一个 zh-TW 分支，作为汉化脚本的繁体版
目前看来可以正常工作
@maboloshi 您看是像现在这样单独弄一个分支还是我自己开一个 Fork 搞更好呢 😊

**Reply by TC999** [reply]
建议单独开分支，然后把安装脚本改成繁中的词库文件就行

毕竟本来仓库名称就包含了繁中（

**Comment by TC999** [comment]
新 BUG：

1.  功能预览具体描述不翻译
   
![image](https://github.com/user-attachments/assets/111460f9-fbd5-4aac-91ac-bfcdcedc9c80)

2.  新版提交详情页，图示位置不翻译，旧版可以
![image](https://github.com/user-attachments/assets/eb6595ce-389d-4396-9174-c23a2acd7138)
![image](https://github.com/user-attachments/assets/b472b969-6949-40bf-86dc-998cb82e9146)

**Reply by wyc-26** [reply]
在我这看所有的功能预览具体描述都不翻译 🤣
是我的问题吗，你们那有翻译吗

**Reply by TC999** [reply]
> 在我这看所有的功能预览具体描述都不翻译 🤣
> 是我的问题吗，你们那有翻译吗

猜测是忽略了

**Reply by maboloshi** [reply]
> > 在我这看所有的功能预览具体描述都不翻译 🤣
> > 是我的问题吗，你们那有翻译吗
> 
> 猜测是忽略了

是的，`markdown-body`规则😢

**Comment by wyc-26** [comment]
![image](https://github.com/user-attachments/assets/a0dcc18e-e7dd-4a07-9bbc-368c3dc93f37)
请教一下大家这个该怎么翻译
> 希望大家永远见不到这个提示（悲

**Reply by TC999** [reply]
```
"Spammy": "仅自己可见",
            "This user is marked as spammy. Their comments will onlyshow in staff mode.": "该用户已被封号，评论仅自己可见。",
```
参考B站控评提示，不完全意译（

**Comment by TC999** [comment]
BUG：像这种按钮，只有第一次加载才会正常翻译，后面更改选项后直接不翻译
![image](https://github.com/user-attachments/assets/9bf774c9-8aae-4ce1-b0b3-3c486c01f584)
![image](https://github.com/user-attachments/assets/f3ac9dac-ff65-471e-83fb-7c92843c37da)

**Reply by maboloshi** [reply]
因为前后两个字段是分离的

**Reply by TC999** [reply]
> 因为前后两个字段是分离的

不是，F5刷新后就正常了

**Reply by TC999** [reply]
其实不止这一处，很多一开始不会出现的词条永远不会翻译，即使词库文件里有对应词条

**Reply by TC999** [reply]
> 其实不止这一处，很多一开始不会出现的词条永远不会翻译，即使词库文件里有对应词条

在1.9.2中是正常的，到1.9.3就有这个问题

**Comment by baojinlaofu** [comment]


**Reply by wyc-26** [reply]
没开开发者模式

**Reply by TC999** [reply]
……文档里写着 **“打开开发者模式”** 后安装脚本

**Comment by LuYifei2011** [comment]
BUG：部分时间未翻译
`具体某条 代码扫描 警报/<user-name>/<repo-name>/security/security/code-scanning/<id>`

<img width="314" height="161" alt="image" src="https://github.com/user-attachments/assets/ce8ae443-8b0e-42ed-9ae5-06d542f421cf" />


<img width="1491" height="186" alt="image" src="https://github.com/user-attachments/assets/8c8f5852-d8ba-408d-bdc2-0ba303748606" />

**Comment by LuYifei2011** [comment]
BUG：部分词条无法翻译

<img width="1349" height="584" alt="image" src="https://github.com/user-attachments/assets/c39c2be3-0329-4824-b5f5-2a0db900986b" />
<img width="1600" height="209" alt="image" src="https://github.com/user-attachments/assets/4dbf78dd-8c8c-47f1-9ff7-e7f0d474e5d6" />
